opensearch/sycamore-opensearch.sh

```diff
@@ -579,6 +669,14 @@ setup_transient() {
     '{"transient":{"cluster":{"metadata":{"aryn_deploy_complete":1}}}}'
 }
 
+model_is_deployed() {
+    local model_id="$1"
+    debug "checking whether ${model_id} is deployed"
+    local model_state="$(_curl "${BASE_URL}/_plugins/_ml/models/${model_id}" | jq -r '.model_state')"
```

Comment:
May want to split this out so we can debug-print the curl output.
```
  local model_info=$(_curl ...)
  [[ -z ${model_info} ]] && return 1
  debug "" "${model_info}"
  local model_state=$(jq -r '.model_state' <<< ${model_info})
```
Can probably drop the other debug then.